src/main/scala/system/ExampleRocketSystem.scala

```diff
@@ -37,6 +37,8 @@ class ExampleRocketSystem(implicit p: Parameters) extends RocketSubsystem
     sbus.coupleTo("coherence_manager") { in :*= _ }
     mbus.coupleFrom("coherence_manager") { _ :=* BankBinder(mbus.blockBytes * (nBanks-1)) :*= out }
   }
+
+  override def getOMGlobalInterruptSignals() = None
```

Comment:
There actually are interrupt signals, in trait `HasAsyncExtInterrupts` which is mixed in here.